Content.Shared/Shipyard/Prototypes/VesselPrototype.cs

```diff
@@ -0,0 +1,52 @@
+using Robust.Shared.Prototypes;
+using Robust.Shared.Serialization;
+using Robust.Shared.Serialization.TypeSerializers.Implementations.Custom.Prototype;
+using Robust.Shared.Utility;
+
+namespace Content.Shared.Shipyard.Prototypes
+{
+    [Prototype("vessel")]
+
+    public sealed class VesselPrototype : IPrototype
+    {
+        [ViewVariables]
+        [IdDataField]
+        public string ID { get; } = default!;
+
+        /// <summary>
+        ///     Vessel name.
+        /// </summary>
+        [ViewVariables]
+        [DataField("name")] public string Name = string.Empty;
+
+        /// <summary>
+        ///     Short description of the vessel.
+        /// </summary>
+        [ViewVariables]
+        [DataField("description")] public string Description = string.Empty;
+
+        /// <summary>
+        ///     The price of the vessel
+        /// </summary>
+        [DataField("price", required: true)]
+        public int Price { get; }
+
+        /// <summary>
+        ///     The prototype category of the product. (e.g. Small, Medium, Large, Emergency, Special etc.)
+        /// </summary>
+        [DataField("category")]
+        public string Category { get; } = string.Empty;
+
+        /// <summary>
+        ///     The prototype group of the product. (e.g. Civilian, Syndicate, Contraband etc.)
+        /// </summary>
+        [DataField("group")]
+        public string Group { get; } = string.Empty;
```

Comment:
Are these prototypes? I don't remember seeing them.

Reply:
this was taken from how Cargo computer handles sorting its product listings? I don't think they are actual prototypes but used in the UI sort instead